packages/itwinui-react/package.json

```diff
@@ -63,8 +63,8 @@
     "clean:coverage": "rimraf coverage",
     "clean": "rimraf .turbo && yarn clean:coverage && yarn clean:build && rimraf node_modules",
     "test": "yarn test:types && yarn test:unit",
-    "test:unit": "jest",
-    "test:unit:watch": "jest --watch",
+    "test:unit": "dotenv -v NODE_OPTIONS=--experimental-vm-modules -- npx jest",
```

Comment:
Since `dotenv` is used in `react-workshop` and `itwinui-react`, should its dependency be defined in the root `package.json`?